pytorch_lightning/overrides/data_parallel.py

```diff
@@ -149,74 +150,65 @@ def parallel_apply(self, replicas, inputs, kwargs):
         return parallel_apply(replicas, inputs, kwargs, self.device_ids[:len(replicas)])
 
 
-class LightningDistributedDataParallel(DistributedDataParallel):
-    """
-    Override the forward call in lightning so it goes to training and validation step respectively
-    """
-    PREPARE_FOR_BACKWARDS = True
+class LightningDistributedWrapper(torch.nn.Module):
```

Comment:
btw, this smells like api change ;]

Reply:
ok, deprecation warning and remove in 1.4?

Reply:
@Borda I also added a deprecation test, but it is not so simple because it needs torch.distributed to be initialized, so it will add about 3-4 seconds only for that test. Don't know how to do it simpler